docs/getting_started/basic_usage/basic_usage.rst

```diff
@@ -115,6 +83,27 @@ network
     e.g. you can choose between different LSTM implementations, or GRU, or standard RNN, etc.
     See :class:`TFNetworkRecLayer.RecLayer` or :class:`NetworkRecurrentLayer.RecurrentUnitLayer`.
     See also :ref:`tf_lstm_benchmark`.
+..
+    batching
+        The sorting variant when the mini-batches are created. E.g. ``random``.
+
+batch_size
+    The total number of frames. A mini-batch has at least a time-dimension
+    and a batch-dimension (or sequence-dimension), and depending on dense or sparse,
+    also a feature-dimension.
+    ``batch_size`` is the upper limit for ``time * sequences`` during creation of the mini-batches.
+
+max_seqs
+    The maximum number of sequences in one mini-batch.
+
+..
+    chunking
```

Comment:
Is that on purpose here, the `..`, and then `chunking` indented?

Reply:
Yes, this is the multiline comment function for Sphinx. Although batching and chunking were in the original file, I think that chunking is a very special parameter and might not be appropriate under basic usage. For batching, I excluded it because I am still not sure if people should rather be encouraged to use "seq_ordering" in the datasets explicitly.

Reply:
`chunking` is very basic. Everyone working with hybrid models would want to use that. This definitely should be in there.

Reply:
I would disagree here, I think at this point only parameters that are needed for ALL models should be mentioned, as otherwise this list needs to be very long. If you say the whole documentation should be focused on ASR, as this is currently its main purpose, then yes it can stay.

Reply:
I would not say focused, but I would say that ASR (and also hybrid models) are still something very important, so I would definitely leave this in for now. Also, this is more generic for all kinds of frame-wise sequence trainings, not just ASR.